scripts/update/01-run.sh

```diff
@@ -12,6 +15,28 @@ echo "=========== Stage: Install ============"
 echo "======================================="
 echo
 
+[[ -f "/etc/default/umbrel" ]] && source "/etc/default/umbrel"
+
+if [[ ! -z "${UMBREL_OS:-}" ]]; then
+    echo 
+    echo "============================================="
+    echo "Installing on Umbrel OS $UMBREL_OS"
+    echo "============================================="
+    echo 
+
+    echo "Copying new release from external storage to a temporary dir on the SD card"
+
+    # Cleanup any tmp dir from unclean install run
+    [[ -d "/home/umbrel/.umbrel-${RELEASE}" ]] && rm -rf "/home/umbrel/.umbrel-${RELEASE}"
+
+    # Copy from external storage to SD card
+    mkdir -p "/home/umbrel/.umbrel-$RELEASE"
+    cp  --recursive \
+        --archive \
+        --no-target-directory \
+        "$UMBREL_ROOT/.umbrel-$RELEASE" "/home/umbrel/.umbrel-$RELEASE"
```

Comment:
Why do we need to copy to this temp directory? "/home/umbrel/.umbrel-$RELEASE"

Can't we just `rsync` directly from `"$UMBREL_ROOT/.umbrel-$RELEASE"` to `$SD_CARD_UMBREL_ROOT `?

Reply:
Ahhh, yes. You're right, this doesn't make any sense now. It was there before because of the mount/unmount logic (that's why an intermediary dir was needed), but since we don't have that problem now, we can just directly copy to the SD card root.

Reply:
Fixed